editoast/src/views/infra/auto_fixes.rs

```diff
@@ -122,6 +124,9 @@ pub enum AutoFixesEditoastError {
     )]
     #[editoast_error(status = 500)]
     MaximumIterationReached(),
+    #[error("Failed during fixes application trial")]
```

Comment:
minor: suggestion (especially since I'm not sure of the English validity of the proposed sentence)
```suggestion
    #[error("Failed trying to apply fixes")]
```

Reply:
I tried to have a sentence that insist on "it's just a trial run".
IMO "trying to" may mislead to "while actually applying".

Anyway, as long as "try" is there I'm OK, so changing for your sentence is possible.

Reply:
Done in https://github.com/osrd-project/osrd/pull/5844/commits/259630ef64bf90c7547a88771ad6174f446c6b03